utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -117,19 +118,24 @@ def should_build(self, host_target):
         return self.args.build_tensorflow_swift_apis
 
     def build(self, host_target):
-        with shell.pushd(self.source_dir):
-            shell.call([
-                os.path.join(self.source_dir, '..', 'swift', 'utils',
-                             'configure-tensorflow-defaults'),
-            ])
-            shell.call([
-                self.toolchain.bazel,
-                "build",
-                "-c", "opt",
-                "--define", "framework_shared_object=false",
-                "//tensorflow:tensorflow",
-            ])
+        # Run the TensorFlow configure script: `yes "" | ./configure`.
+        configure_script_path = os.path.join(self.source_dir, 'configure')
+        yes_process = subprocess.Popen(['yes', ''], stdout=subprocess.PIPE)
```

Comment:
Note: I think that it's possible that upstream Apple folks won't like this, as they would like all filesystem interactions to go through their `shell` wrappers. (Or so I've been told.)

Reply:
Thanks. I tried creating a `shell` wrapper for `subprocess.Popen` but reverted it because it didn't seem worthwhile for this ad-hoc usage.

Progress at https://github.com/dan-zheng/swift/tree/tensorflow-merge-fix-macosx-wip in case we want to pursue it.